### PR TITLE
[TRAFODION-1733][TRAFODION-1734] License text improvements

### PR DESCRIPTION
**Pull request by svarnau**
These changes are due to review feedback received from release 1.3.

    [TRAFODION-1734] License info improvements from reviews of 1.3 release

    Added info on particular type of BSD & MIT licenses used.
    Removed .txt file suffixes.
    Made copy of license rather than give URL.
    Removed license for protobuf, since we merely use the tool rather than
    include their software.

    [TRAFODION-1733] Remove unneeded info from license file

    The FBstring.h file is Apache license, not a different license, so
    no need to call it out.

    ASCIIdoc is a tool that is used to generate files in build process
    but the software is not distributed with Traodion software, so it
    does not apply.

### CODE REVIEW

**Comment by DaveBirdsall** [comment]
Looks good to me.

File: LICENSE [hunk 1]
```diff
@@ -307,11 +286,6 @@ For details see http://jquery.org/license
```
Comment (robertamarton) [inline LICENSE]: Do you need to change this to point to the license file you copied to licenses directory?

File: LICENSE [hunk 2]
```diff
@@ -307,11 +286,6 @@ For details see http://jquery.org/license
 
 ++++++++++++++++++++++++++++++
 
-Apache Trafodion uses asciidocs to create its documentation which is licensed
-under MIT 
-
-For details see http://asciidoctor.org/
```
Comment (robertamarton) [inline LICENSE]: Just to confirm - an original comment we received for our first release attempt (that did include docs) was that we needed to add asciidocs to our license file.  When we released 1.3.0, we did not include docs so we then received a comment that this text was not needed.  Are we going to include docs in our source tree?  If so, will we need this text?

File: LICENSE (hunk 1 above)
Comment (svarnau) [inline LICENSE]: Agreed. I'll fix that oversight.

File: LICENSE (hunk 2 above)
Comment (svarnau) [inline LICENSE]: No, I don't think so. The docs do not include any ASCIIdoc software. It is merely used as a build tool to generate other formats (html, pdf) from asciidoc formatted files.

**Comment by svarnau** [comment]
jenkins, retest -- I don't think sqstart is dependent on the license file!

**Comment by DaveBirdsall** [comment]
Is this ready for commit?

**Comment by svarnau** [comment]
If Roberta is satisfied with my response to her comments, then yes ready to go.

**Comment by robertamarton** [comment]
Go for it.



   roberta



*From:* DaveBirdsall [mailto:notifications@github.com]
*Sent:* Friday, February 5, 2016 4:21 PM
*To:* apache/incubator-trafodion <incubator-trafodion@noreply.github.com>
*Cc:* Roberta Marton <roberta.marton@esgyn.com>
*Subject:* Re: [incubator-trafodion] [TRAFODION-1733][TRAFODION-1734]
License text improvements (#307)



Is this ready for commit?

—
Reply to this email directly or view it on GitHub
<https://github.com/apache/incubator-trafodion/pull/307#issuecomment-180632754>
.

**Pull request merged**